src/components/molecules/SpaceEditFormNG/SpaceEditFormNG.tsx

```diff
@@ -229,7 +229,7 @@ export const SpaceEditFormNG: React.FC<SpaceEditFormNGProps> = ({
 
   const backButtonOptionList = ownedVenues.filter(
```

Comment:
This could probably benefit from `useMemo()` to not re-render the component using the array. Bonus, the `?? ALWAYS_EMPTY_ARRAY` can be incorporated here so `backButtonOptionList` is always an array.